crates/ursa-network/src/behaviour.rs

```diff
@@ -116,6 +121,7 @@ where
     pub fn new<B: BitswapStore<Params = P>>(
         keypair: &Keypair,
         config: &NetworkConfig,
+        origin_config: &OriginConfig,
```

Comment:
This is replicated behaviour, you already have access to origin_config through `NetworkConfig`

Reply:
It's added as a top level config, not under network config. I could switch origin config under network config but I had thought that it made more readable as it's own instead of nested

Reply:
You still have crate access to it. `config::OriginConfig`

Reply:
Not sure what that solves, NetworkConfig and OriginConfig are separate entities passed in from the ursa bin

Reply:
Not sure exactly what you mean, Origin and Network Configs are separate entities passed from the ursa binary

Reply:
OriginConfig can be part of network config as a param or something contained.

Reply:
refactored implementation into rpc and moved it to `[server_config.origin]`
https://github.com/fleek-network/ursa/pull/278/commits/11e9610901aad513e46cbede8f05907d30ba1363